Community-Aktivitäten/README.md

```diff
@@ -94,6 +94,23 @@ Dauer: 3 Std.
 Ausstattung: In unserem Workshop soll wirklich gearbeitet werden, deshalb bringt am besten einen Laptop (macOS, Linux, Windows) mit, einer JVM/Kotlin/Gradle fähigen Entwicklungsumgebung in der aktuellen Version. Das ist zum Beispiel IntelliJ oder Android Studio. Aktuelle Informationen zum Workshop gibs hier: https://michalharakal.github.io/KPTChat
 
 
+## Tech2Talk - Erstelle mit Hilfe erfahrener Speaker deinen ersten eigenen Vortrag! (JUG Ostfalen: Steven Schwenke, JUG Darmstadt: Falk Sippach und Sergej Tihonov)
+
+Beschreibung: Vorträge zu halten ist in der IT Community eine der wichtigsten Mittel, Wissen zu verteilen. Zusätzlich dazu erhöht der Sprecher seine eigene Sichtbarkeit und wird als Experte in seinem Thema wahrgenommen. Das sind nur zwei der vielen Vorteile eines Konferenz- oder User Group Votrages.
+
+In diesem Community-Event gehst du Hands-On mit Hilfe von gleich drei sehr erfahrenen Speakern die ersten Schritte auf dem Weg zu deinem ersten Vortrag. Gemeinsam finden wir dein ganz individuelles Thema, strukturieren deinen Vortrag und bereiten dich umfassend auf deinen Auftritt vor. Anschließend werfen wir noch einen Blick auf die oft vernachlässigte Nachbereitung und auf Materialien, die dir dabei helfen, ein erfolgreicher Speaker zu werden. 
+
+Der ganze Workshop ist so aufgebaut, dass die Teilnehmer und ihre entstehenden Talks im Vordergrund stehen. Wir nehmen uns viel Zeit, ausführlich Fragen zu beantworten und bereits gesammelten Erfahrungen einfließen zu lassen. Am Ende hast du nicht nur ganz konkrete inhaltliche Ansätze für deinen ersten eigenen Vortrag, sondern weißt auch genau, wie du dich organisatorisch vorbereiten solltest. Im besten Fall hast du sogar schon die erste Version deines Vortrags für einen ersten Probelauf!
+
+[Tech2Talk](https://tech2talk.de) ist eine von Falk Sippach, Sergej Tihonov und Steven Schwenke gegründete Community-Organisation, die in der Technologie-Branche arbeitende Personen dabei unterstützt, ihr Wissen mittels Vorträgen zu teilen. So fördert sie eine lebendige Community, den Austausch untereinander, persönliche Weiterentwicklung und aktives Lernen.
```

Comment:
Werbung ist leider bei CAs nicht zugelassen. Bitte entfernt den Hinweis auf tech2talk.de, oder entfernt alle kommerziellen Angebot von jener Seite. Danke.

Reply:
Das ist jetzt Quatsch. Tech2Talk ist eine rein ehrenamtliche Seite. Dass wir dort Bücher empfehlen, die Leute zusätzlich zu unseren Angeboten anschauen können, ist aus meiner Sicht keine Werbung. Wenn es hilft, nehme ich gern noch 3 andere Buchempfehlungen online, womit andere Leute theoretisch Geld verdienen. Durch Bücher wird man meines Wissens nicht sehr reich. 

Genaugenommen war das Buch sogar zuerst da und wir haben daraus die Community abgeleitet. Um aber etwas von dem Buchtitel wegzukommen, haben wir uns vor ein paar Wochen zu dritt einen Tag in unserer Freizeit mit einem Rebranding beschäftigt, wodurch jetzt der Name tech2talk rausgekommen ist. 

Bei dem Buch hat Steven übrigens keine kommerziellen Hintergedanken, obwohl er da einiges an Zeit und Geld (Lektorat, Layout) investiert hat. In den letzten beiden Jahren hatten wir ja bereits eine ähnliche Community Aktivität in gleicher Konstellation, bei Steven den Teilnehmer sogar eine kostenfreie Kopie des Buchs gegeben hat. Und wir investieren unabhängig von der JavaLand mehrmals jährlich unsere Zeit in kostenlose Online-Coachings, bei denen wir schon 10+ JavaLand Newcomer unterstützt haben.

Reply:
Falk, das mag alles sein, aber leider weiß der 0815-Leser der Beschreibung das alles nicht, und wir haben in den vergangenen Jahren schon mehrfach Anfeindungen hinsichtlich kommerziell / Eigenwerbung abwehren müssen (wir haben sogar von CA-Einreichern erlebt dass sie wegen solcher Gedanken zurückgezogen haben). Ich würde Dich daher wirklich bitten, den Verweis auf die Seite einfach zu streichen, um uns nicht angreifbar zu machen. CAs sind nicht für Werbung da, auch nicht für ein Ehrenamt. Danke.

Reply:
Ok, ich verstehe. Dann sehe ich zwei Möglichkeiten: Ich baue eine eigene Landing-Page für die CA. Wäre es ok, wenn das auf tech2talk.de/javaland läuft und dort das Buch nicht verlinkt ist? Alternativ kann ich es bei der JUG Darmstadt bzw. CyberLand anlegen.

Reply:
Klar kannst Du beides gerne so machen. Oder halt alternativ einfach *im Beschreibungstext* den Verweis zur Tech2Talk-Initiative einfach weglassen.

Reply:
Ich habe nun den Text von Tech2Talk um Referenzen zu unserer Initiative bereinigt. Eventuell wird @sippsack nochmal einen PR aufmachen mit einer entsprechend gestalteten Website.

Zusätzlich habe ich das angefragte Mentoring Speed-Dating aufgenommen.

Hoffentlich können wir diesen PR damit nun mergen.